_includes/accessibility-tests/wcag-details.html

```diff
@@ -122,6 +122,11 @@
   {% assign criterion_name = "Location" %}
   {% assign criterion_url = "https://www.w3.org/TR/WCAG21/#location" %}
 {% endif %}
+{% if item.wcag_criterion == "2.4.9" %}
+  {% assign criterion_level = "AAA" %}
+  {% assign criterion_name = "Location" %}
```

Comment:
**issue:** This criterion name is incorrect

```suggestion
  {% assign criterion_name = "Link purpose (link only)" %}
```

Reply:
Resolved in [5b6b02e](https://github.com/uswds/uswds-site/pull/2863/commits/5b6b02e37396e05112ca7be66889a42238fad443)